### ISSUE TITLE
Some questions about training custom dataset

### ISSUE THREAD

**Issue by LansyMorndye** [gyy456/CityGS-X#9]
Thank you very much for your excellent work. However, I had various errors when trying to train my own data(Only UAV RGB image data). I wonder if you could publish a complete process for training on the customer dataset? I'm pretty believe it will help a lot in this job.

**Comment by gyy456** [comment]
What is your training data structure like?  If it's different from the structure in Readme, you may have to change the dataloader code.   The RGB data and colmap data are needed. If you don't want to load depth images. You can set '--not_use_dpt_loss'.

**Comment by LansyMorndye** [comment]
Thanks for your kindly reply. I used RGB images only and did colmap first followed by 3DGS, which generate such folders as follow.

<img width="717" alt="Image" src="https://github.com/user-attachments/assets/77fc764c-9fd3-462c-b056-1b206a0015d9" />

Then I ignored the depth information and ran following command:
CUDA_VISIBLE_DEVICES=0 torchrun --standalone --nnodes=1 --nproc-per-node=1 train.py --bsz 8 -s /datasets/czx/Data/test_1 --resolution 4 --model_path ./output/test1_citygsx --iterations 100000 --not_use_dpt_loss --not_use_multi_view_loss --not_use_single_view_loss 

The output path shown as follow:

<img width="1398" alt="Image" src="https://github.com/user-attachments/assets/581a6ce4-2f7f-4809-90bd-b480891763e5" />

However, the .ply file cannot open in viewer, supersplat and other Visualization App. I wonder if there is any way to visualize my result.

**Comment by gyy456** [comment]
You may have to download the **Octree-GS viewer** for visualization.

**Comment by LansyMorndye** [comment]
I notice that the Octree-GS viewer seem to be the same as SIBR viewers? I wonder if we can communicate in more efficient way such as wechat(wechat name: LansyMorndye)?

**Comment by gyy456** [comment]
ok

**Comment by DYL857857** [comment]
Could you please tell me how your "matrixcity_areial" dataset was processed? Based on the process of cityGaussianV2, I used the following command to prepare the data using the generated COLMAP results. The script data_proc_mc.sh shows that there are many missing images in the test set. Moreover, the official provided COLMAP results do not match the images in the test set.
